infrastructure/kzg/src/test/java/tech/pegasys/teku/kzg/KZGTest.java

```diff
@@ -260,6 +260,17 @@ public void incorrectTrustedSetupFilesShouldThrow(final String path) {
     assertThat(cause.getMessage()).contains("Failed to parse trusted setup file");
   }
 
+  @ParameterizedTest(name = "trusted_setup={0}")
+  @ValueSource(strings = {"mainnet/trusted_setup_old.txt", "minimal/trusted_setup_old.txt"})
+  public void oldTrustedSetupFilesShouldThrow(final String path) {
```

Comment:
I would rename test to `loadingTrustedSetupInMonomialFormShouldThrow()` and rename suffix of the files to `_monomial` instead of `_old`.

Reply:
Done, renamed the files and the test

Reply:
nice one, thanks